pkg/jsonlog/jsonlog.go

```diff
@@ -6,12 +6,20 @@ import (
 	"time"
 )
 
+// JSONLog decsribes a JSONLog.
```

Comment:
It would be helpful to put what a JSONLog is, what it is used for.
Think about a newcomer reading and trying to figure it out.